tests/integrations/tso/go.mod

```diff
@@ -123,10 +122,10 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/power-devops/perfstat v0.0.0-20221212215047-62379fc7944b // indirect
-	github.com/prometheus/client_golang v1.11.1 // indirect
-	github.com/prometheus/client_model v0.2.0 // indirect
-	github.com/prometheus/common v0.26.0 // indirect
-	github.com/prometheus/procfs v0.6.0 // indirect
+	github.com/prometheus/client_golang v1.19.0 // indirect
```

Comment:
Is it necessary to update their version?

Reply:
Yes, It's necessary. because we relay the new Interface `DeletePartialMatch`  in client_golang v1.19.